code/data_structures/src/list/singly_linked_list/operations/insertion/insertion_at_nth_position.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+  C++ program to insert element at the nth position of a Singly Linked List
+  Part of Cosmos by OpenGenus Foundation
+ */
+#include<iostream>
+#include<cstdlib>
+
+using namespace std;
```

Comment:
Instead of this, you may use std:: prefix. The reason is the size of the namespace associated :+1: